README.md

```diff
@@ -428,6 +428,17 @@ machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Verify that the node labels have been configured correctly**
```

Comment:
This note is not so useful for users, the major step missing here is what they should do if they couldn't find those labels?

Instead, I would prefer to mention the setup steps for explicitly for 1.13 cluster:
- Enable needed featureGates : CSINodeInfo, CSIDriverRegistry
- Install needed CRDs

Reply:
true, but as @pohly mentioned, we have kept back from documenting gates-add process as there are many variants. And we already list the needed gates in README, so it would be repetition here. CRD, that we need to add, but is it in this chapter?

Reply:
> true, but as @pohly mentioned, we have kept back from documenting gates-add process as there are many variants.

We should mention as a note that - cluster should be enabled needed feature gates, but we can leave out how to do that.

Reply:
pushed new state adding note that gates are needed, and example commands how to install CRDs

Reply:
Full sentence, end with a dot here.

Reply:
?? but that would make this bulleted line different from all others. I dont feel it deserves a dot

Reply:
Then let's leave it out for now. Perhaps @MCamp859 can do a pass over the document to check this.